File: GGAndroid_2.0/GGControls/GGViewModel/src/main/java/io/ganguo/viewmodel/ui/dialog/BottomVModelDialog.java
package io.ganguo.viewmodel.ui.dialog;

import android.content.Context;
import android.databinding.ViewDataBinding;

import io.ganguo.vmodel.BaseViewModel;

/**
 * <p>
 * 底部弹窗 - dialog
 * </p>
 * Created by leo on 2018/9/30.
 */
public abstract class BottomVModelDialog<T extends ViewDataBinding, B extends BaseViewModel> extends FullScreenVModelDialog<T, B> {
    public BottomVModelDialog(Context context) {
        super(context);
    }

}

File: GGAndroid_3.0/core/frame/mvvm-v1/src/main/java/io/ganguo/viewmodel/pack/ui/dialog/BottomVModelDialog.kt
package io.ganguo.viewmodel.pack.ui.dialog

import android.content.Context
import androidx.databinding.ViewDataBinding
import io.ganguo.viewmodel.core.BaseViewModel

/**
 *
 *
 * 底部弹窗 - dialog
 *
 * Created by leo on 2018/9/30.
 */
abstract class BottomVModelDialog<T : ViewDataBinding, B : BaseViewModel<*>>(context: Context, theme: Int) : FullScreenVModelDialog<T, B>(context, theme)